### PR TITLE
add OpenSSF best practices and FAIR rating badge

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [ ] Tests for the changes have been added (for bug fixes / features)
  - [ ] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [ ] HISTORY.rst has been updated (with summary of main changes)
  - [ ] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added
- [ ] `bumpversion patch` has been called on this branch
- [ ] The relevant author information has been added to `.zenodo.json`

### What kind of change does this PR introduce?

* Adds a badge that showcases our rating with the OpenSSF
* Adds a FAIR rating badge (generated with https://github.com/fair-software/howfairis)

### Does this PR introduce a breaking change?

No.

### Other information:

For our security badge, see: https://bestpractices.coreinfrastructure.org/en/projects/6041
For the FAIR software principles, see: https://fair-software.eu/

The security rating is not currently passing, however there are compliance checks that will require time and effort. Security rating is at around 88% and can easily increase by at least 7-8% with minimal effort and time. Full compliance should be possible during the summer.

Our repo badges are getting out of hand. I think I might try organizing them when I have the chance this summer.

### CODE REVIEW

**Review by huard — COMMENTED** [review]
I suggest we wait until you complete the form before merging.

**Comment by Zeitsperre** [comment]
@huard 

I've added in the remaining information. This leaves our rating at around 88%. There are a few things I can do to push us closer to 96%, but it will need to be done in the coming months. As far as I'm concerned, this is a good indicator for now. I leave it to you to merge.

**Review by huard — APPROVED** [review]

**Pull request merged**
